br/pkg/lightning/restore/restore.go

```diff
@@ -1236,7 +1255,54 @@ func (rc *Controller) buildRunPeriodicActionAndCancelFunc(ctx context.Context, s
 
 var checksumManagerKey struct{}
 
+const (
+	pauseGCTTLForDupeRes       = time.Hour
+	pauseGCIntervalForDupeRes  = time.Minute
+	pauseGCServiceIDForDupeRes = "lightning-duplicate-resolution"
+)
+
+func (rc *Controller) pauseGCOnce(ctx context.Context, pdCli pd.Client) error {
+	// Get current GC safe point.
+	safePoint, err := pdCli.UpdateGCSafePoint(ctx, 0)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// Pause GC for 1 hour.
+	_, err = pdCli.UpdateServiceGCSafePoint(ctx,
+		pauseGCServiceIDForDupeRes, int64(pauseGCTTLForDupeRes/time.Second), safePoint)
+	return err
+}
+
+func (rc *Controller) keepPauseGC(ctx context.Context, pdCli pd.Client) {
+	ticker := time.NewTicker(pauseGCIntervalForDupeRes)
```

Comment:
```suggestion
	ticker := time.NewTicker(pauseGCIntervalForDupeRes/3)
```
We should use a smaller interval to ensure the lease is updated on time. `pauseGCIntervalForDupeRes / 3` is a typically good value to tolerate occational failure.

Reply:
`pauseGCIntervalForDupeRes` is not the ttl 😂, ttl is `pauseGCTTLForDupeRes`.